.github/workflows/cmake_build.yaml

```
@@ -18,3 +18,7 @@ jobs:
         uses: threeal/cmake-action@main
         with:
           cxx-compiler: ${{ matrix.compiler }}
+      - name: Build tests
+        run: cd build && make mdio_acceptance_test  && make mdio_variable_test && make mdio_dataset_test
+      - name: Run tests
+        run: cd build/mdio/  && ./mdio_acceptance_test && ./mdio_variable_test && ./mdio_dataset_test
```

```suggestion
      - name: Build tests
        run: cd build && make -j
      - name: Run tests
        run: cd build/mdio/  && ./mdio_acceptance_test && ./mdio_variable_test && ./mdio_dataset_test && ./mdio_dataset_factory_test && ./mdio_dataset_validator_test && ./mdio_stats_test && ./mdio_utils_test && ./mdio_variable_collection_test
```
There are a handful of optional cloud-store tests as well, but those shouldn't be part of the CICD pipelines.